Documentation/gettingstarted/clustermesh.rst

```diff
@@ -335,6 +335,34 @@ Cilium will automatically perform load-balancing to pods in both clusters.
      selector:
        name: rebel-base
 
+Load-balancing with Shared Services
+###################################
+
+SharedService if set to false, prevents a service from being shared,
+the default is true if GlobalService is set, otherwise false,
+Setting the annotation SharedService to false while setting
+GlobalService to true allows to expose remote endpoints without
+sharing local endpoints.
```

Comment:
I had to read this several times to try to get it, which probably suggests that we can improve on the wording a bit to make this more user-friendly. I've suggested an alternative text below, do you think this is more clear?

```suggestion
By default, a Global Service will load-balance across backends in multiple clusters.
This implicitly configures ``io.cilium/shared-service: "true"``. To prevent service
backends from being shared to other clusters, and to ensure that the service
will only load-balance to backends in remote clusters, this option should be
disabled.
```

/cc also @tgraf to confirm that the wording is precise here.

Reply:
Yeah, this is better to understand in one read.